capture/screenshot.js

```diff
@@ -26,20 +26,19 @@ const screenshot = async (url) => {
 }
 
 const downloadImage = async (url) => {
```

Comment:
cmiiw, i think u dont need this `async` prefix on this function, because on this function nothing process to async call (await), but only and already purely returns a promise

Reply:
I forgot to remove async thanks dude 😁